README.md

```diff
@@ -63,7 +63,7 @@ $ curl --request GET "http://localhost:9922/metrics"
 
 # HELP smb_metrics_status Current metrics-collector status versions
 # TYPE smb_metrics_status gauge
-smb_metrics_status{commitid="092fe2bb0",ctdbvers="4.20.0-103",sambaimage="",sambavers="4.20.0-103",version="v0.2-28-g092fe2b"} 1
+smb_metrics_status{commitid="092fe2bb0",ctdbvers="4.20.0-103",netbiosname="cluster1",sambaimage="",sambavers="4.20.0-103",version="v0.2-28-g092fe2b"} 1
```

Comment:
The Dashboard team would likely prefer a label such as cluster_name or cluster_id I guess, as `netbiosname` may not be a feasible option for them. Anyways let's check with them itself

Reply:
We provide a solution which should work form and Samba deployment. It the specific case of ceph+smb deployment `cluster_id` will be mapped to `netbios name`.

Reply:
make sense!